apps/test/unit/lib/levelbuilder/CourseOfferingEditorTest.jsx

```diff
@@ -122,7 +126,9 @@ describe('CourseOfferingEditor', () => {
         grade_levels: 'K,1,2,3',
         curriculum_type: 'Course',
         header: 'Self-Paced',
-        marketing_initiative: 'HOC'
+        marketing_initiative: 'HOC',
+        cs_topic: 'Art and Design',
+        school_subject: 'Science,English Language Arts'
```

Comment:
these will need to be updated to use the non-translatable values, e.g. `science`, `english_language_arts` ...